<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textRandomBaseNumber">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>随即计数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="NowMode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>英译中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chineseToEnglishMode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>中译英</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ListenMode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>听力练习</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="randomMode">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>乱序词典</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>当前模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="listMode">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>输入顺序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dictionaryMode">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>字典序</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>序列模式：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="englishToChineseMode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>英译中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="inputMode">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>88</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>输入顺序</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
